src/member.cc

```diff
@@ -228,11 +264,13 @@ class RedisChainModule {
   std::string next_address_;
   std::string next_port_;
 
-  MasterClient master_client_;
+  std::unique_ptr<MasterClient> master_client_;
```

Comment:
Just curious, any reason you changed this to a pointer?

Reply:
The compiler told me not to allocate an interface type. I take this to mean that a pointer type is needed to support then indirection I added between MasterClient and RedisMasterClient.